src/app/modules/main/chat_section/chat_content/messages/module.nim

```diff
@@ -290,6 +293,7 @@ method messageAdded*(self: Module, message: MessageDto) =
                     message.transactionParameters.transactionHash,
                     message.transactionParameters.commandState,
                     message.transactionParameters.signature),
+    message.mentionedUsersPks
```

Comment:
you should get an indentation error when building ?

Reply:
Nope, it does not belong to newTransactionParametersItem(). It is another parameter in initItem() :
![image](https://user-images.githubusercontent.com/61889657/159451085-06dc685b-2322-4ae7-a7f9-fbfbfe9f6fa9.png)